pages/commit/[sha].tsx

```diff
@@ -102,21 +99,9 @@ function VersionControlLinks({
 
 const fetcher = (url: string) => fetch(url).then((res) => res.json());
 
-function CommitInfo({ sha }: { sha: string }) {
-  const { data: commit } = useSWR(`/api/commit/${sha}`, fetcher, {
-    refreshInterval: 60 * 1000, // refresh every minute
-    // Refresh even when the user isn't looking, so that switching to the tab
-    // will always have fresh info.
-    refreshWhenHidden: true,
-  });
-  if (commit === undefined) {
-    return <div>Loading...</div>;
-  }
-
+export function CommitData({ commit }: { commit: CommitData }) {
```

Comment:
Can we pull this out into `components/` instead of exporting it directly from here?

Reply:
Sure. And thanks for the pointers to the other fields. Didn't know they were there.

I think we should eventually add auth to this and just pull from GH though. Like if a person updates the PR title, would it get logged into Rockset?

Reply:
yes, all events from GH are recorded so the information is guaranteed to be accurate (within replication latency). 

It's fine to set up auth and pull from GH. Although it does make things more complicated, since we can't use GH for the main HUD query due to rate limiting constraints, which means we would have to maintain two data sources.

Reply:
I think the only technical disadvantage from getting everything from rockset is right now there is ~3-5m replication lag between an event happening in GH and it getting reflected into rockset. This could be mitigated by using dynamo as our webhook payload store instead of s3 (which would decrease the replication lag to ~5 seconds) but I am too lazy to do it